### PR TITLE
Use new Copy icon

### PR DESCRIPTION
**Pull request by rickycodes**
Update copy-to-clipboard to use new tx-history icon added in #8180

I noticed we're not using the tooltip for the copy button in the contact list, we probably should? I can add that if we want it in another PR

### CODE REVIEW

File: ui/app/pages/settings/contact-list-tab/view-contact/view-contact.component.js [hunk 1]
```diff
@@ -55,11 +56,9 @@ export default class ViewContact extends PureComponent {
               >
                 { quadSplit(checkSummedAddress) }
               </div>
-              <img
-                className="address-book__view-contact__group__static-address--copy-icon"
-                onClick={() => copyToClipboard(checkSummedAddress)}
-                src="/images/copy-to-clipboard.svg"
-              />
+              <button className="address-book__view-contact__group__static-address--copy-icon" onClick={() => copyToClipboard(checkSummedAddress)}>
```
Comment (Gudahtt) [inline ui/app/pages/settings/contact-list-tab/view-contact/view-contact.component.js]: Nit: could you put each of these props on a new line? It requires horizontal scrolling when viewing on GitHub at the moment.

**Review by Gudahtt — COMMENTED** [review]
Just checking, but are these icons both the same size as before? 🤔 

The SVG being replaced had a height of 17 and a width of 18, but in CSS it was set to 30px on the contact list but 10px on the tx details - I'm not sure which one won out. The new one is 20px but I know it's not exactly the same, so maybe the end-result is around the same size?

**Comment by rickycodes** [comment]
> Just checking, but are these icons both the same size as before? 
> 
> The SVG being replaced had a height of 17 and a width of 18, but in CSS it was set to 30px on the contact list but 10px on the tx details - I'm not sure which one won out. The new one is 20px but I know it's not exactly the same, so maybe the end-result is around the same size?

They're about the same size now:

image

And here's what the contacts one looks like:

image

I'll break those two props out onto their own lines.

**Review by Gudahtt — DISMISSED** [review]
LGTM!

**Comment by rickycodes** [comment]
I actually noticed the icon on the contacts page needs a bit of padding at certain widths...

**before:**

image

**after:**

image

**Review by Gudahtt — APPROVED** [review]
Good catch!

**Pull request merged**